example/flask/server.py

```diff
@@ -128,21 +162,22 @@ def notifywithmetadata():
             }
         },
     )
-    return ('Metadata was added to the notification, check <a href=\"bugsnag.com\">bugsnag.com</a> ' +
-        ' to view on the "Request info" and "Resolve info" tabs')
+    return ('Metadata was added to the notification, check' +
```

Comment:
Without a space this get smushed into the anchor tag

```suggestion
    return ('Metadata was added to the notification, check ' +
```